File: python-version/LC3392/count_subarrays.py
class Solution:
    @staticmethod
    def countSubarrays(nums: list[int]) -> int:
        ans = 0
        n = len(nums)
        for i in range(1, n-1):
            if nums[i] == (nums[i-1]+ nums[i+1]) * 2:
                ans += 1
        return ans

File: go-version/leetcode/LC3392/count_subarrays.go
package LC3392

/**
 * @Author: cutejiuge cutejiuge@163.com
 * @Date: 2025/4/27 上午8:28
 * @FilePath: count_subarrays
 * @Description: 遍历模拟即可
 */

func countSubarrays(nums []int) int {
	var ans int
	n := len(nums)
	for i := 1; i < n-1; i++ {
		if nums[i] == (nums[i-1]+nums[i+1])*2 {
			ans++
		}
	}
	return ans
}
